sdks/java/io/mongodb/src/main/java/org/apache/beam/sdk/io/mongodb/MongoDbIO.java

```diff
@@ -180,6 +194,14 @@ private MongoDbIO() {}
 
       abstract Builder setNumSplits(int numSplits);
 
+      abstract Builder setLimit(int limit);
+
+      abstract Builder setPipeline(List<BsonDocument> pipeline);
+
+      abstract Builder setDocumentIdStr(String documentId);
+
+      abstract Builder setDocumentId(ObjectId documentId);
```

Comment:
Maybe makes more sense to be consistent and call all refs of this objectId and the other one just documentId

Reply:
Agreed.